less/index.less

```diff
@@ -54,6 +54,65 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
+	
 }
 
+// ======== === === == =====Variables
+@width: 10px;
+@height: @width + 10px;
+
+
+// header {
+// 	width:@width;
+// 	height: @height;
+// }
+// variables
+
+@color:lime;
+// @bodyFontStack: font-family:"Supria Sans", Helvetica, Arial, sans-serif;
+
+header {
+	nav {
+		.centerflex;
+		.border;
+	}
+}
+
+body {
+	@media (max-width:500px) {
```

Comment:
your media query should be nested better than this, but I'll give you credit for having it in here